packages/amplify-graphql-searchable-transformer/src/cdk/create-searchable-datasource.ts

```diff
@@ -5,7 +5,7 @@ import { ResourceConstants } from 'graphql-transformer-common';
 import assert from 'assert';
```

Comment:
I see a few uses of `ElasticsearchDataSourceLogicalID` in this file. I think we could replace those as well by adding `OpenSearchDataSourceLogicalID` to the `graphql-transformer-common` module.